docs/interfaces.md

```diff
@@ -12,198 +12,362 @@ Slots may support multiple connections to plugs.  For example the OS snap
-allows adjusting settings of other applications.
+Can access global gsettings of the user's session which gives privileged access
+to sensitive information stored in gsettings and allows adjusting settings of
+other applications.
 
-Usage: reserved
-Auto-Connect: yes
+* Auto-Connect: yes
+* Availability: OS snap (classic)
 
-### optical-drive
+### home
 
-Can access the first optical drive in read-only mode. Suitable for CD/DVD playback.
+Can access non-hidden files in user's `$HOME` and gvfs mounted directories
+owned by the user to read/write/lock.
 
-Usage: common
-Auto-Connect: yes
+* Auto-Connect: yes on classic, no on native
+* Availability: OS snap
 
 ### mpris
 
-Can access media players implementing the Media Player Remote Interfacing
-Specification (mpris) when the interface is specified as a plug.
+Providing snaps implementing the Media Player Remove Interfacing Specification
+(mpris) may be accessed via their well-known DBus name.
 
-Media players implementing mpris can be accessed by connected clients when
-specified as a slot.
+Consuming snaps can access media players implementing mpris via the providing
+snap's well-known DBus name.
 
-Usage: common
-Auto-Connect: no
+* Auto-Connect: no
+* Availability: with providing snap
 
-### camera
+### network
 
-Can access the first video camera. Suitable for programs wanting to use the
-webcams.
+Can access the network as a client.
+
+* Auto-Connect: yes
+* Availability: OS snap
+
+### network-bind
+
+Can access the network as a server.
+
+* Auto-Connect: yes
+* Availability: OS snap
+
+### opengl
+
+Can access OpenGL hardware.
+
+* Auto-Connect: yes
+* Availability: OS snap (classic)
+
+### optical-drive
+
+Can access the first optical drive in read-only mode. Suitable for CD/DVD
+playback.
+
+* Auto-Connect: yes
+* Availability: OS snap (classic)
+
+### pulseaudio
+
+Can access the PulseAudio sound server which allows for sound playback in games
+and media application. Recording not supported but will be in a future release.
+
+* Auto-Connect: yes
+* Availability: OS snap (classic)
+
+### unity7
+
+Can access Unity7. Unity 7 runs on X and requires access to various DBus
+services. This interface grants privileged access to the user's session since
+the Unity 7 environment does not prevent eavesdropping or apps interfering with
+one another.
+
+* Auto-Connect: yes
+* Availability: OS snap (classic)
 
-Usage: common
-Auto-Connect: no
+### x11
+
+Can access the X server which gives privileged access to the user's session
+since X does not prevent eavesdropping or apps interfering with one another.
+
+* Auto-Connect: yes
+* Availability: OS snap (classic)
 
 ## Supported Interfaces - Advanced
 
+### bluez
+
+Can access snaps providing the bluez interface which gives privileged access to
+bluetooth.
+
+* Auto-Connect: no
+* Availability: with providing snap
+
+### bool-file
+
+Can access GPIO paths for LED brightness and GPIO values.
```

Comment:
We'll see an actual gpio interface soon (it's being worked on right now..). Can we please drop docs for bool-file for the time being? Not sure we'll want to keep it.